### PR TITLE
Bff strapi last

### PR DESCRIPTION
**Pull request by eduard-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -163,7 +163,7 @@ export const live_chat_key = 'live_chat_redirection'
 export const reddit_url = 'https://www.reddit.com/user/Deriv_official/'
 export const telegram_url = 'https://t.me/derivdotcomofficial'
 export const youtube_url = 'https://www.youtube.com/@deriv'
-export const loss_percent = 72
+export const fallback_loss_percent = 80
```
Comment (habib-deriv) [inline src/common/constants.ts:166]: We can't have fallback loss percentage
```suggestion
export const fallback_loss_percent = 80
```

Comment (eduard-deriv) [reply src/common/constants.ts]: thanks, fixed

File: src/pages/who-we-are/index.tsx [hunk 2]
```diff
@@ -37,24 +39,190 @@ const EndSeparator = styled.div`
...
+                        url
+                        publicURL
+                    }
+                }
+            }
+            slider_medias {
+                slider_1 {
+                    localFile {
+                        publicURL
+                    }
```
Comment (hasan-deriv) [inline src/pages/who-we-are/index.tsx:194]: We can create graphql fragment for repeatable fields.

File: src/pages/who-we-are/_types.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
+type StrapiImage = {
+    localFile: {
+        publicURL: string
```
[the remark this answers is not in the document]
Comment (NikitK-deriv) [reply src/pages/who-we-are/_types.ts:3]: It's already in avif format @hasan-deriv

File: src/pages/who-we-are/_MakeTrading.tsx [hunk 4]
```diff
@@ -30,24 +32,32 @@ const StyledFirstSectionText = styled(Header)`
...
     font-weight: 400;
+
     @media ${device.laptop} {
         padding: 0 0 24px 0;
     }
 `
-
-const MakeTrading = () => {
+type TFirstSectionTexts = {
+    text: string
```
[the remark this answers is not in the document]
Comment (NikitK-deriv) [reply src/pages/who-we-are/_MakeTrading.tsx:41]: nope, cms for content managers first
they don't know about our technical _t_ implementation
but, translation still works

File: src/pages/who-we-are/index.tsx (hunk 2 above)
Comment (NikitK-deriv) [reply src/pages/who-we-are/index.tsx:194]: Good point 
Screenshot 2023-08-30 at 09 05 27

File: src/common/constants.ts [hunk 5]
```diff
@@ -163,7 +163,6 @@ export const live_chat_key = 'live_chat_redirection'
 export const reddit_url = 'https://www.reddit.com/user/Deriv_official/'
 export const telegram_url = 'https://t.me/derivdotcomofficial'
 export const youtube_url = 'https://www.youtube.com/@deriv'
-export const loss_percent = 72
```
Comment (NikitK-deriv) [inline src/common/constants.ts:166]:
```suggestion
export const loss_percent_fallback = 72
```

File: src/features/components/molecules/layout-overlay/cfd-warning-banner/index.tsx [hunk 6]
```diff
@@ -15,6 +15,14 @@ const CfdWarningBanner = () => {
...
     const { is_eu, is_cpa_plan } = useRegion()
     const [expanded, setExpanded] = useState(false)
+    const data = useStaticQuery(graphql`
+        query {
+            strapiCfdWarningBanner {
+                loss_percent
+            }
+        }
+    `)
+    const loss_percent = data?.strapiCfdWarningBanner?.loss_percent
```
Comment (NikitK-deriv) [inline src/features/components/molecules/layout-overlay/cfd-warning-banner/index.tsx:25]:
```suggestion
    const loss_percent = data?.strapiCfdWarningBanner.loss_percent || fallback
```

File: src/features/components/templates/navigation/market-nav/index.tsx [hunk 7]
```diff
@@ -7,10 +8,14 @@ import { market_nav_logo } from './market-nav.module.scss'
...
-import { usePlatformQueryParam } from 'components/hooks/use-platform-query-param'
 
 const MarketNav = () => {
-    const { is_deriv_go } = usePlatformQueryParam()
+    const [platform] = useQueryParam('platform', StringParam)
+    const [is_deriv_go, setIsDerivGo] = useState(false)
+
+    useEffect(() => {
+        setIsDerivGo(platform === 'derivgo')
+    }, [platform])
```
Comment (NikitK-deriv) [inline src/features/components/templates/navigation/market-nav/index.tsx:18]: I believe it's not a part of bff task

**Review by NikitK-deriv — COMMENTED** [review]
I didn't found integration with crowdin
we need to make a graphql data fetch (check original version of branch) and make some changes in pre-commit script

File: src/pages/who-we-are/index.tsx (hunk 2 above)
Comment (eduard-deriv) [reply src/pages/who-we-are/index.tsx:194]: @hasan-deriv there are too many problems with using a fragment in this case, starting with typing, continuing with Graphql validation, ending with the inability to determine and find this fragment during assembly, I tried, it works too unstable for such a minor improvement.

**Review by hasan-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by konstantinosG-deriv — APPROVED** [review]

**Pull request closed without merging**